spyder/plugins/ipythonconsole.py

```diff
@@ -816,13 +816,27 @@ def get_plugin_actions(self):
         """Return a list of actions related to plugin."""
         create_client_action = create_action(
                                    self,
-                                   _("Open an &IPython console"),
+                                   _("Open a &new console"),
                                    icon=ima.icon('ipython_console'),
                                    triggered=self.create_new_client,
                                    context=Qt.WidgetWithChildrenShortcut)
         self.register_shortcut(create_client_action, context="ipython_console",
                                name="New tab")
 
+        create_pylab_action = create_action(
+                                   self,
+                                   _("Open a new &pylab console"),
+                                   icon=ima.icon('ipython_console'),
+                                   triggered=self.create_pylab_client,
+                                   context=Qt.WidgetWithChildrenShortcut)
+
+        create_sympy_action = create_action(
+                                   self,
+                                   _("Open a new &sympy console"),

```

Comment:
Remove the ampersand and capitalize `sympy`.

Reply:
To be clear, proper capitalization is ``SymPy``